contents/handbook/people/team-structure/platform.md

```diff
@@ -9,28 +9,62 @@ hideAnchor: true
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
-## Mission
+# Platform Sub-Team Mission Statements
 
+## Ingestion
 
-Make deploying, scaling and managing PostHog easy
+**Provide the best events pipeline in the world.**
 
-## Goals & Key Results
+## Infrastructure
 
-* PostHog is the easiest self-hosted product to deploy and scale in the world
-    * Result: Volume of clients requiring help @marcushyett-ph
-    * Result: % of successful installs / upgrades @guidoiaquinti
-    * Result: Max size of operating instances @fuziontech
-    * Monitor: % of users on latest version of PostHog @tiina303
-    * Result: We don't lose customers because we can't scale enough (shared with Core Analytics) @marcushyett-ph
-* Move all posthog-supported instances to self-hosted
-    * Result: No alerts come to us from customer-instances
-* All valid events are ingested correctly
-    * Result: Time ingestion is down on cloud (from alert) @fuziontech
-    * Result: Lost events between ingress and clickhouse @fuziontech
-* Data is as up to date as possible
-    * Result: Latency between ingress and clickhouse @fuziontech
-* A good experience for plugin developers
-    * Result: Good docs, alerting, debugging and testing @yakkomajuri
+**Make deploying, scaling, and managing PostHog easy, fast, and reliable.**
+
+# Roadmap
+
+## Ingestion
+
+### 3 year
+
+- Ingest events out of order
+- No events left behind (99.99%)
+- All events are correct (99.99%)
+- The pipeline scales perfectly linearly and intercept is low (smallest instance runs on $5 node)
+- Events are ingested and visible in app within 5 seconds p99
```

Comment:
Why is this goal important? Why not 1 minute or 10 minutes?

People are not using posthog as a real-time data system AFAIK.

Reply:
It's a side effect of the experience and expectation that people like seeing events come in in near real time.

The most important use case for near real time ~5 second p99 latency is for the onboarding flow and for instrumentation of your app. If you are onboarding and you can't get past the onboarding flow where we help you instrument your app for the first event because of pipeline latency that is a 'bad time' ™️. It also helps as a product engineer responsible for instrumenting your app further seeing events show up quickly for instant feedback that you instrumented the product correctly to emit events to PostHog.